docs/csharp/tuples.md

```diff
@@ -426,6 +426,20 @@ if (("Althea", "Goodwin") == p)
 
 The `Deconstruct` method could convert the `Person` object `p` to a tuple containing two strings, but it is not applicable in the context of equality tests.
 
+## Tuples as out parameters
+
+Tuples can be used as out parameters _themselves_. Not to be confused with any ambiguity previously mentioned in the [Deconstruction](#deconstruction) section. In a method call, you need only describe the tuple's shape:
+
+[!code-csharp[TuplesAsOutParameters](../../samples/snippets/csharp/tuples/program.cs#01_TupleAsOutVariable "Tuples as out parameters")]
```

Comment:
By convention, we use the `~` on samples (I know other examples don't yet, we're updating links as we update articles):

```suggestion
[!code-csharp[TuplesAsOutParameters](~/samples/snippets/csharp/tuples/program.cs#01_TupleAsOutVariable "Tuples as out parameters")]
```

Reply:
Good to know--looking forward to helping docs an samples with more PR's!